apps/web/components/NewPipelineInScreen.tsx

```diff
@@ -1,202 +0,0 @@
-import {useAtom} from 'jotai'
```

Comment:
This is unused. Delete to allow us to replace the old `Tabs` with the ones updated with the current design.